roles/confluent.test/molecule/mtls-custombundle-rhel/verify.yml

```diff
@@ -104,15 +104,6 @@
         property: ssl.keystore.location
         expected_value: /var/ssl/private/ksql.keystore.jks
 
-    - name: Secrets Protection Test
```

Comment:
@domenicbove why is this not required ?

Reply:
I've updated the test molecule.yml variables to have 
```
secrets_protection_enabled: true
ksql_secrets_protection_enabled: false
```
IE no props should be masked for ksql. A better test would actually be to validate that ksql in fact does not have this property masked (rather than removing this task). I'll make that change

Reply:
thanks. it will be great to ensure ksql configs are masked as well. Ping me when you have those, so I can review. Other changes in this PR LGTM.

Reply:
Ok! ya ill go back to how it was